javascript/package.json

```diff
@@ -10,7 +10,8 @@
     "scripts": {
         "test": "mocha",
         "only-test": "mocha --inspect-brk test/test_refactor.js",
-        "prettier": "prettier --write './{src,test}/**/*.js'"
+        "prettier": "prettier --write './{src,test}/**/*.js'",
+        "format": "prettier --check './{src,test}/**/*.js'"
     },
```

Comment:
Minor suggestion - can we have more helpful names for these? It is not immediately clear which of `format` and `prettier` is a check and which actually formats code. Maybe `format` can be renamed to `check_format`.

Reply:
Done https://github.com/uber/piranha/pull/84/commits/eb3b0ae92f2aa59d565d45e5487d9a9512f1724a